runtime/vm/CRIUHelpers.cpp

```diff
@@ -456,13 +456,30 @@ initializeCriuHooks(J9VMThread *currentThread)
 		}
 	}
 
+#if JAVA_SPEC_VERSION >= 20
+	{
+		J9VMSystemProperty *vtParallelism = NULL;
+		PORT_ACCESS_FROM_VMC(currentThread);
+		if (J9SYSPROP_ERROR_NONE != getSystemProperty(vm, "jdk.virtualThreadScheduler.parallelism", &vtParallelism)) {
```

Comment:
we need to make sure this is done after processing the restore args

Reply:
Since user can set it at startup and on restore so we need to detect both.

Reply:
`jdk.virtualThreadScheduler.parallelism` is read via [private static final ForkJoinPool DEFAULT_SCHEDULER = createDefaultScheduler()](https://github.com/ibmruntimes/openj9-openjdk-jdk20/blob/4b0d1648f68805474340a94cd7b3f173bc6c83a4/src/java.base/share/classes/java/lang/VirtualThread.java#L76) at VM startup.
`j.l.VirtualThread.ForkJoinPool.parallelism` won't be affected by the system property specified by the restore option file.
Added a test to confirm this behaviour.